assets/js/translator.js

```diff
@@ -132,6 +133,12 @@ if(modeEnabled('translation')) {
             }, timeout);
 
             synchronizeTextareaHeights();
+
+            if(isURL($('#originalText').val())) {
+                setTimeout(function () {
+                    translateWebpage();
```

Comment:
The timer for webpage translation will fire irrespective of whether the `Instant Translation` checked or not. The one for translateText triggers only when it is checked.